### PR TITLE
security: fix seclink header

### PR DESCRIPTION
**Pull request by pillip8282**
fix seclink header

### CODE REVIEW

**Review by jeongchanKim — APPROVED** [review]
LGTM

File: os/include/tinyara/seclink_drv.h [hunk 1]
```diff
@@ -15,6 +15,8 @@
  * language governing permissions and limitations under the License.
  *
  ****************************************************************************/
+#pragma once
```
Comment (sangam-swami) [inline os/include/tinyara/seclink_drv.h:18]: why can't we use standard method #ifndef __SECLINK_DRV_H__  ?

Comment (pillip8282) [reply os/include/tinyara/seclink_drv.h:18]: I think pragma once is easy to read.

**Review by an4967 — APPROVED** [review]
LGTM

**Pull request merged**